app/config/security.yml

```diff
@@ -9,19 +11,24 @@ security:
     providers:
         in_memory:
             memory:
-                users: %security_users%
+                # Only for demo purposes. Of course, use a more
+                # secure password on production.
+                users:
+                    anna_admin: { password: kitten, roles: ROLE_ADMIN }
```

Comment:
* removed the user account, it's not used anywhere in the demo.
* added `anna_admin/kitten`, as this is equal to the one used in the Symfony demo.
* kept `admin/admin`, to not confuse people already used to use this account in the demo.